src/invidious/views/components/player.ecr

```diff
@@ -167,7 +167,6 @@ if (bpb) {
 <% end %>
 
 
-var video = player;
```

Comment:
`videojs-youtube-annotatations.js` relies on the player being called `video`. Without, it throws:
```
video is not defined
```
[here](https://github.com/afrmtbl/videojs-youtube-annotations/blob/master/dist/dist.js#L408).

Reply:
Oh... oops!

Reply:
Updated dist.js with a fixed copy.